src/api/css.ts

```
@@ -2,6 +2,39 @@ import { type AnyNode, type Element, isTag } from 'domhandler';
+ * @returns The name as it appears in a `style` attribute.
+ */
+function hyphenate(name: string): string {
+  if (name.startsWith('--')) return name;
+  // CSSOM uses cssFloat (IE: styleFloat) for the float property.
+  if (name === 'cssFloat' || name === 'styleFloat') return 'float';
+
+  const hyphenated = name.replace(rUpperCase, (char) => `-${char.toLowerCase()}`);
+  // CSSOM spellings like webkitTextStrokeWidth omit the leading dash.
+  // Only do that for camelCase input. An already-hyphenated name such as
+  // webkit-text-stroke-width or a non-vendor name that merely starts with
+  // o-/ms- after folding must stay as written.
```

ESLint

[error] Expected a block comment instead of consecutive line comments.

(multiline-comment-style)